src/chrome/content/rules/Linode.xml

```diff
@@ -1,32 +1,67 @@
 <!--
-	Problematic hosts in *linode.com:
+	Nonfunctional hosts in *linode.com:
+
+		- speedtest.atlanta ¹
+		- speedtest.dallas ¹
+		- speedtest.frankfurt ¹
+		- speedtest.fremont ¹
+		- speedtest.london ¹
+		- speedtest.newark ¹
+		- speedtest.singapore ¹
+		- speedtest.tokyo ¹
 
-		- status *
+		- mirrors ¹
+		- atlanta.mirrors ¹
+		- dallas.mirrors ¹
+		- frankfurt.mirrors ¹
+		- fremont.mirrors ¹
+		- london.mirrors ¹
+		- newark.mirrors ¹
+		- singapore.mirrors ¹
+		- tokyo.mirrors ¹
 
-	* StatusPage.io
+		- *.members ²
+		- *.nodebalancer ²
 
+	¹ Refused
+	² Customer load balancers and VPSes. Mismatched, refused or time out.
 
-	Fully covered hosts in *linode.com:
+	Problematic hosts in *linode.com:
 
-		- (www.)?
-		- blog
-		- forum
-		- library
-		- manager
-		- stats
-		- status	(→ linode.statuspage.io)
+		- apt ¹
+		- status ²
 
+	¹ "gnutls_handshake() warning: The server name sent was not recognized"
+	  SSL Labs says "Incorrect SNI alerts: apt.linode.com"
+	  It works in Chrome and Firefox, though.
```

Comment:
@J0WI It means the HTTPS vhost isn't configured with "ServerAlias apt.linode.com" (though it works because it's the default vhost and the cert is a wildcard), so the server sends back an `unrecognized_name` warning alert:

    $ openssl s_client -connect apt.linode.com:443 -servername apt.linode.com -state
    CONNECTED(00000003)
    SSL_connect:before/connect initialization
    SSL_connect:unknown state
    SSL3 alert read:warning:unrecognized name
    SSL_connect:SSLv3 read server hello A
    ...

According to the obsolete [RFC 4366 § 3.1](https://tools.ietf.org/html/rfc4366#section-3.1):

       If the server understood the client hello extension but does not
       recognize the server name, it SHOULD send an "unrecognized_name"
       alert (which MAY be fatal).

According to the current [RFC 6066 § 3](https://tools.ietf.org/html/rfc6066#section-3):

    If the server understood the ClientHello extension but
       does not recognize the server name, the server SHOULD take one of two
       actions: either abort the handshake by sending a fatal-level
       unrecognized_name(112) alert or continue the handshake.  It is NOT
       RECOMMENDED to send a warning-level unrecognized_name(112) alert,
       because the client's behavior in response to warning-level alerts is
       unpredictable.

Evidently, the tests are choosing to treat it as fatal, while Chrome and Firefox ignore it.

I came across a wget thread and patch about changing their GnuTLS code from the former to the latter:

https://lists.gnu.org/archive/html/bug-wget/2013-05/msg00033.html

I dunno if the test stuff can do the same.